src/bfs.h

```diff
@@ -0,0 +1,52 @@
+/* Breadth First Search Implementation using Adjacency List 
+ *	representation of the graph
+ *
+ * Example:
+ *	#include "bfs.h"
+ *
+ *	int main() {
+ *		struct Graph *G;
+ *		struct attributes *n;
```

Comment:
In this example, you should pass a `malloc`ed list. This is a cython independent example.